config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show create] do
```

Comment:
[REQUIRED] Don't forget to include new to your resources. Since your new action will need it once you create it

Reply:
Well resolved ✔️